content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -39,31 +39,37 @@ model where `Pods` can be treated much like VMs or physical hosts from the
 perspectives of port allocation, naming, service discovery, load balancing,
 application configuration, and migration.
 
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address and MAC address.
+This means that containers within a `Pod` can all reach each other's ports on
+`localhost`. This also means that containers within a `Pod` must coordinate port
+usage, but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (which is an optional capability). Even if a host have
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, so the "IP-per-pod" model is guaranteed.
+
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods on a node can communicate with all pods on all nodes without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+   * pod-network pods on a node can communicate with all pod-network pods on
+     all nodes without NAT
+   * non-pod processes on a node (e.g. system daemons, kubelet) can communicate
+     with all pods on that node
```

Comment:
nit: we [avoid Latin abbreviations](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases)

Reply:
I'm not sure this is correct for Windows. I had thought that the situation there was that kubelet can communicate with all pods on the node, but no other system processes can reach any pods. I guess we need clarification/confirmation here.

Reply:
@jsturtevant can you help us here?

Reply:
+1 to @jsturtevant comment.

Reply:
that is the expected and how this started, the decision was to only require communications from services to pods on the same node, across nodes is not required